sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -683,6 +712,7 @@ public void run() {
 
     @WorkerThread
     private synchronized void handleCheckForUpdate() {
+        mManualCheckForUpdateRequested = true;
```

Comment:
Is it okay to assign `true` even though `tryResetWorkflow()` returns `false`?

Reply:
I guess I don't understand why it wouldn't be...